packages/react/src/themes/teams/components/Chat/chatMessageStyles.ts

```diff
@@ -82,13 +71,12 @@ const chatMessageStyles: ComponentSlotStylesInput<
     position: 'absolute',
     right: v.actionMenuPositionRight,
     top: v.actionMenuPositionTop,
-    overflow: p.focused ? 'visible' : 'hidden',
 
-    // hide and squash actions menu to prevent accidental hovers over its invisible area
-    opacity: p.focused ? 1 : 0,
-    width: p.focused ? 'auto' : 0,
+    visibility: 'hidden',
```

Comment:
This is the only source of truth about whether the action menu should be opened.

Reply:
I'm affraid this will break the accessibiity for JAWS, I will test in short